src/lib/component/SettingDialog/template/index.js

```diff
@@ -0,0 +1,135 @@
+      value="${autocompletionWs}">
+  </div>
+  <div class="textae-editor__setting-dialog__details">
+    <details>
+      <summary>Delimiter Characters</summary>
+      <table>
+        <tr class="textae-editor__setting-dialog__delimiter-character-add-row">
+          <td><input class="textae-editor__setting-dialog__delimiter-character-add-input" type="text"></td>
+          <td><button class="textae-editor__setting-dialog__delimiter-character-add-button">+</button></td>
+        </tr>
+        ${() =>
+          delimiterCharacters
+            .slice()
+            .reverse()
+            .map(
+              (char) => `
+              <tr class="textae-editor__setting-dialog__delimiter-character-row">
+                <td>
+                  <input
+                    type="text"
+                    class="textae-editor__setting-dialog__delimiter-character-input"
+                    value="${escapeForDisplay(char)}">
+                </td>
+                <td><button class="textae-editor__setting-dialog__delimiter-character-delete-button">&times;</button></td>
+              </tr>`
+            )
+            .join('')}
+      </table>
+    </details>
+  </div>
+  <div class="textae-editor__setting-dialog__details">
+    <details>
+      <summary>Non-edge Characters</summary>
+      <table>
+        <tr class="textae-editor__setting-dialog__blank-character-add-row">
+          <td><input class="textae-editor__setting-dialog__blank-character-add-input" type="text"></td>
+           <td><button class="textae-editor__setting-dialog__blank-character-add-button">+</button></td>
+        </tr>
+        ${() =>
+          blankCharacters
+            .slice()
+            .reverse()
+            .map(
+              (char) => `
+              <tr class="textae-editor__setting-dialog__blank-character-row">
+                <td>
+                  <input
+                    type="text"
+                    class="textae-editor__setting-dialog__blank-character-input"
+                    value="${escapeForDisplay(char)}">
+                </td>
+                <td><button class="textae-editor__setting-dialog__blank-character-delete-button">&times;</button></td>
+              </tr>`
+            )
+            .join('')}
+      </table>
+    </details>
+  </div>
+  <div class="textae-editor__setting-dialog__details">
+    <details>
+      <summary>Function Availability</summary>
+      <div class="textae-editor__setting-dialog__function-availability-list">
+        ${() =>
+          functionAvailability.names
+            .map(
+              (name) => `
+              <label class="textae-editor__setting-dialog__function-availability-label">
+                <input
+                  type="checkbox"
+                  class="textae-editor__setting-dialog__function-availability-checkbox"
+                  ${functionAvailability.isAvailable(name) ? `checked="checked"` : ''}>
+                ${name}
+              </label>`
```

Comment:
関数に抜き出して欲しいです。

```suggestion
              (name) => toHogeHogeElement(name)
```

のイメージです。

Reply:
修正しました。
https://github.com/pubannotation/textae/pull/173/commits/7943936e4ede3bce003ed4a3fb53aa3646c6777b

指摘箇所と合わせて、delimiter/non-edge characterの部分も関数に切り出しました。